openstudiocore/src/model/AirflowNetworkDistributionLinkage_Impl.hpp

```diff
@@ -91,14 +91,23 @@ namespace detail {
     /** @name Setters */
     //@{
 
-    // TODO: Check argument type. From object lists, some candidates are: AirflowNetworkNodeAndZone.
-    bool setNode1(const AirflowNetworkNode& airflowNetworkNodeAndZone);
-
-    // TODO: Check argument type. From object lists, some candidates are: AirflowNetworkNodeAndZone.
-    bool setNode2(const AirflowNetworkNode& airflowNetworkNodeAndZone);
-
-    // TODO: Check argument type. From object lists, some candidates are: AirflowNetworkComponent.
-    bool setComponent(const AirflowNetworkComponent& airflowNetworkComponent);
+    template <typename T> bool setNode1(const T& airflowNetworkNode)
```

Comment:
Does this have to be a template?  I'm unclear why 

    bool setNode1(const AirflowNetworkNode& airflowNetworkNodeAndZone);

would not work?